src/lib/stores/api.js

```diff
@@ -130,12 +130,13 @@ export default {
         switch (e.type) {
           case "rally-sdk.complete-signup":
             // @ts-ignore
-            const studyId = e.detail;
-            if (!studyId) {
+            if (!e.detail && !e.detail.studyId) {
```

Comment:
Slight optimization:
```
studyId = e.detail && e.detail.studyId;

if (!studyId) {
...
}
```

Reply:
OK thanks, I kept as exception to make it consistent with elsewhere... we should discuss assert vs. exceptions